Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -278,7 +278,14 @@ extension BrowserViewController: WKNavigationDelegate {
 
                 let requestUA = navigationAction.request.value(forHTTPHeaderField: "User-Agent") ?? ""
                 if UserAgent.isDesktop(ua: requestUA) != tab.desktopSite {
-                    webView.load(URLRequest(url: url, cachePolicy: .reloadIgnoringLocalCacheData, timeoutInterval: 60) as URLRequest)
+                    let _url: URL
+                    if ["m.", "mobile."].contains(where: { url.host?.starts(with: $0) ?? false }) {
```

Comment:
Do you think we can make this work with wikipedia?
Wikipedia uses `en.m.wikipedia.org `

Reply:
Good point, fixing...